pkg/btf/validation.go

```diff
@@ -261,3 +262,54 @@ func validateSycall(kspec *v1alpha1.KProbeSpec, name string) error {
 
 	return nil
 }
+
+func GetSyscallsYaml(notMatch bool) (string, error) {
+	crd := `apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "syscalls"
+spec:
+  kprobes:`
+
+	filter := `
+    selectors:
+    - matchBinaries:
+      - operator: "In"
+        values:
+        - "/krava"`
```

Comment:
Can we add a flag for this as well?

E.g., 

```
tetra tracingpolicy generate all-syscalls --match-binaries /krava
```

Reply:
ok, will add